content/en/methods/oauth.md

```diff
@@ -1,17 +1,13 @@
 ---
-title: oauth API methods
+title: OAuth API methods
```

Comment:
this was intentionally not capitalized, i think?

Reply:
I don't see why? It's the title of the page?